README.md

```diff
@@ -64,29 +121,18 @@ The example DAG can be exercised in one of two ways:
 1. Initialize a project with the [Astro CLI](https://www.astronomer.io/docs/cloud/stable/develop/cli-quickstart)
 2. Copy the example DAG into the `dags/` folder of your astro project
 3. Copy the directories in the `include` folder of this repository into the `include` directory of your Astro project
-4. Copy your GCP `credentials.json` file into the base directory of your Astro project
-5. Add the following to your `Dockerfile` to install the `airflow-provider-great-expectations` package, enable xcom pickling, and add the required Airflow variables and connection to run the example DAG:
+4. Add the following to your `Dockerfile` to install the `airflow-provider-great-expectations` package, and add the required Airflow variables and connection to run the example DAG:
```

Comment:
Could we clarify what connection is required to run the example DAG? I’m unsure if our new operators rely on those connections, so I’d like to understand more.

Reply:
thanks - this isn't necessary right now, so i'll remove. Right now, the provider doesn't use any connections, and relies on env variables in order to connect to cloud. I plan to write a GXCloudConnection as followup work.